tests/evaluate/test_tbrom.py

```diff
@@ -779,9 +779,16 @@ def test_generate_snapshot_with_tbrom_is_ok(self):
         # TODO LUCAS - Use another twin model with named selection smaller than whole model
         ns = twinmodel.get_named_selections(romname)
         snp_vec_ns = twinmodel.generate_snapshot(romname, False, named_selection=ns[0])
-        assert len(snp_vec_ns) == 313266
-        assert np.isclose(snp_vec_ns[0], 1.7188266861184398e-05)
-        assert np.isclose(snp_vec_ns[-1], -1.3100502753567515e-05)
+        assert (
```

Comment:
Could you simplify line 782 to 789 by assert `snp_vec_ns.shape[0] == 313266`?
and same below for all assert statements using `len()` on a `numpy.ndarray`?

Reply:
changes done (note that most of these tests were implemented in previous branches/PR)